src/locales/uk/translation.json

```diff
@@ -30,6 +31,7 @@
   "contactUs": "Напишіть нам",
   "contacts": "Контакти",
   "continueWith": "Продовжити з",
+  "copyrights": "©2020 – Data Ocean. Всі права захищені.",
```

Comment:
delete this pls

Reply:
Actually i was delete it, IDK why the translation still was there.